Src/Support/GoogleApis.Auth/OAuth2/Flows/GoogleAuthorizationCodeFlow.cs

```diff
@@ -40,6 +41,12 @@ public class GoogleAuthorizationCodeFlow : AuthorizationCodeFlow
         /// <summary>Gets or sets the include granted scopes indicator.</summary>
         public bool? IncludeGrantedScopes { get { return includeGrantedScopes; } }
 
+        private readonly IEnumerable<KeyValuePair<string, string>> userDefinedQueryParams;

```

Comment:
A reminder to set up a test for GoogleAuthorizationCodeFlow with the new user defined query params.